language/tools/cost-synthesis/src/stack_generator.rs

```diff
@@ -205,12 +207,14 @@ where
                 peek.expect("[Next Integer] Unable to cast peeked stack value to a u8."),
             )
         } else {
-            self.gen.gen_range(0, u8::max_value())
+            self.gen.gen_range(1, 16u8)
```

Comment:
why 16?

Reply:
Going to hazard a guess: Because we don't want to have overflow if we multiply two `u8`s together -- 16 * 16 = `u8::max_value()`

Reply:
Exactly. We do the same thing for u64 and u128 by only generating numbers up to half the max width.